docs/designers-developers/designers/menu-item.md

```diff
@@ -0,0 +1,151 @@
+# MenuItem
+
+MenuItem is a component which renders a button intended to be used in combination with the DropdownMenu Component.
+
+![An image of a MenuItem being highlighted inside of a DropdownMenu component](https://wordpress.org/gutenberg/files/2018/11/MenuItem.png)
+
+1. MenuItem
+
+
+
+## Table of contents
+
+
+
+1. [Design guidelines](http://#design-guidelines)
+2. [Development guidelines](http://#development-guidelines)
+3. [Related components](http://#related-components)
+
+
+
+##Design guidelines
```

Comment:
This heading (and the followings) are missing a space after the `##`

Reply:
Whoops! Thank you, will update.